torch/csrc/profiler/kineto_shim.h

```diff
@@ -12,18 +12,47 @@
 #undef USE_KINETO
 #endif
 
-#include <ActivityType.h>
-
 #include <torch/csrc/Export.h>
 #include <torch/csrc/profiler/api.h>
 
 #ifdef USE_KINETO
+#include <ActivityType.h>
+
 // Forward declarations so we don't have to include `libkineto.h` in a header.
 namespace libkineto {
 class GenericTraceActivity;
 struct CpuTraceBuffer;
 class ActivityTraceInterface;
 } // namespace libkineto
+
+#else
+namespace libkineto {
+// Should match
+// https://github.com/pytorch/kineto/blob/main/libkineto/include/ActivityType.h
+enum class ActivityType {
```

Comment:
Yeah I understand this might not be a good idea. 

It's possible to refactor the profiler to not rely on this enum, but that's beyond me and, perhaps not a good idea either with the added complexity.